test/presenters/supergroups/supergroup_test.rb

```diff
@@ -0,0 +1,28 @@
+require 'test_helper'
+
+include RummagerHelpers
+
+describe Service do
```

Comment:
If you are testing the Superclass here, I'd rename the test to describe `Supergroup`. Otherwise, if you are testing `Service` I'd combine this test with serrvice_test.

Reply:
Thanks.